src/main.rs

```diff
@@ -269,6 +269,15 @@ macro_rules! eat {
         }
     };
 }
+macro_rules! tryc {
```

Comment:
I know you just moved this macro out of the `fn find_ram_in_linker_script`, but can you please add a doc-comment explaining what it is doing?

```suggestion
/// add doc-comment here
macro_rules! tryc {
```